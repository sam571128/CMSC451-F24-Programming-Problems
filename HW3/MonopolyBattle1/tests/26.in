26 64 609
4 7
9 24
24 20
16 19
4 24
3 12
17 23
19 3
19 5
22 11
12 22
1 10
8 26
11 16
16 2
21 18
24 3
10 26
10 2
19 19
15 24
2 13
21 26
24 24
12 10
10 21
9 18
8 9
18 3
18 7
17 6
20 5
23 21
3 15
19 14
14 4
23 6
20 7
18 1
25 5
14 3
19 15
23 15
8 16
5 19
8 12
10 9
5 24
26 26
23 18
6 10
9 9
22 7
10 15
8 17
18 12
2 21
20 16
3 13
7 18
22 16
5 17
4 5
16 22
