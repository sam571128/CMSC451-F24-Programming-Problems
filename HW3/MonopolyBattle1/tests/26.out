39 29 29 36 29 62 30 -1 47 29 29 29 29 35 34 29 62 29 29 34 29 29 62 34 -1 29 